controllers/lvmcluster_controller.go

```diff
@@ -74,7 +88,11 @@ func (r *LVMClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		// Error reading the object - requeue the request.
 		return ctrl.Result{}, err
 	}
-
+	err = r.checkIfOpenshift(ctx)
+	if err != nil {
+		//Ignore the error for now
```

Comment:
```suggestion
		// Ignore the error for now
```

Reply:
why ignore the error?  error should be propagated if we can't detect the platform.

Reply:
For non-Openshift clusters it should still work so take a chance and go ahead?